resources/js/src/app/components/common/LazyMountIntersect.vue

```diff
@@ -0,0 +1,76 @@
+<script>
+    export default {
+        props:
+        {
+            threshold: {
+                type: Number,
+                default: 0.1
+            }
+        },
+
+        data()
+        {
+            return {
+                isVisible: false,
+                loadingRendered: false
+            }
+        },
+
+        created()
+        {
+            this.featureEnabled = 'IntersectionObserver' in window;
+
+            if(!this.featureEnabled)
+            {
+                this.isVisible = true;
+                return;
+            }
+
+            this.observer = new IntersectionObserver((entries) => {
+                if(entries[0].intersectionRatio >= this.threshold && this.loadingRendered)
+                {
+                    this.observer.unobserve(this.$el);
+                    this.isVisible = true;
+                }
+            }, {
+                root: null,
+                rootMargin: '0px',
+                threshold: 0.1
```

Comment:
```suggestion
                threshold: this.threshold
```
...wenn du es schon als Prop bereitstellst

Reply:
Es gibt einen Unterschied zwischen den beiden thresholds.

Das Threshold, welches wir als Option an den Intersection Observer geben, sagt an, wie viel "Visibility Change" passieren muss, dass er das Callback triggert. Das Threshold, welches wir als Prop eingeben, sagt aus, wie viel Visibility das observierte Element haben muss, um den Mount zu starten.

Ich habe die IntersectionObserver spezifische Threshold extra auf 0.1 gesetzt, da bei sehr schnellen scrollen und hohem Threshold es passieren kann, dass der Callback nicht triggert.
Deshalb würde ich diesen Change Request nicht empfehlen.

Reply:
Da du immer nur ein Element beobachtest, sollten beide Werte nach meinem Verständnis identisch sein. Umgekehrt führen alle Werte der Property > 0.1 dazu, dass die Elemente niemals sichtbar werden: Der Observer führt den Callback beim Überschreiten der Sichtbarkeit von 10% einmalig aus. Wenn innerhalb des Callbacks nichts passiert, weil ich hier z.B. einen Threshold von 0.5 angegeben habe, wird der Callback danach nicht wieder ausgeführt. Wenn Werte > 0.1 keinen Sinn machen, dann solltest du das natürlich entsprechend absichern.